pyrr/matrix44.py

```diff
@@ -360,6 +362,30 @@ def create_orthogonal_projection_matrix(
         (Tx, Ty, Tz, 1.),
     ), dtype=dtype)
 
+@all_parameters_as_numpy_arrays
+def create_look_at(eye, target, up, dtype=None):
+    """Creates a look at matrix according to OpenGL standards.
+
+    :param numpy.array eye: Position of the camera in world coordinates.
+    :param numpy.array target: The position in world coordinates that the
+        camera is looking at.
+    :param numpy.array up: The up vector of the camera.
+    :rtype: numpy.array
+    :return: A look at matrix that can be used as a viewMatrix
+    """
+
+    forward = vector.normalise(eye - target)
+    side = vector.normalise(np.cross(up, forward))
+    up = vector.normalise(np.cross(forward, side))
+
+    return np.array((
+            (side[0], up[0], forward[0], 0.),
+            (side[1], up[1], forward[1], 0.),
+            (side[2], up[2], forward[2], 0.),
+            (-np.dot(side, eye), -np.dot(up, eye), -np.dot(forward, eye) , 1.0)
+        ), dtype=dtype)
```

Comment:
Lines 381 -> 386 looks strange to me.

My [code](https://github.com/cprogrammer1994/Headless-rendering-with-python/blob/master/matrices.py#L16) is a bit different.

I will create a few tests then we will see.